extensions/jdbc/jdbc-mssql/runtime/src/main/java/io/quarkus/jdbc/mssql/runtime/MsSQLServiceBindingConverter.java

```diff
@@ -13,6 +13,6 @@ public class MsSQLServiceBindingConverter implements ServiceBindingConverter {
     @Override
     public Optional<ServiceBindingConfigSource> convert(List<ServiceBinding> serviceBindings) {
         return ServiceBinding.singleMatchingByType("sqlserver", serviceBindings)
-                .map(new DatasourceServiceBindingConfigSourceFactory.Jdbc());
+                .map(new DatasourceServiceBindingConfigSourceFactory.Jdbc("jdbc:%s://%s%s;databaseName=%s"));
```

Comment:
Not related to your patch but this is really brittle and close to unreadable. But let's not slow down your patch because of that as it's preexisting.

I would rather have to implement a method that takes proper parameters with names.